README.md

```diff
@@ -1,39 +1,52 @@
 # AWS IoT Device SDK for Java v2
 
-This document provides information about the AWS IoT device SDK for Java V2. This SDK is built on the [AWS Common Runtime](https://docs.aws.amazon.com/sdkref/latest/guide/common-runtime.html)
+The AWS IoT Device SDK for Java v2 connects your Java applications and devices to the AWS IoT platform. It handles the complexities of secure communication, authentication, and device management so you can focus on your IoT solution. The SDK makes it easy to use AWS IoT services like Device Shadows, Jobs, Fleet Provisioning, and Commands.
 
-*__Jump To:__*
+**Supported Platforms**: Linux, Windows 10+, macOS 10.15+, Android API level 24+ (Android 7.0+)
 
+*__Topics:__*
+* [Features](#features)
 * [Installation](#installation)
-* [Android](./documents/ANDROID.md)
+  * [Minimum Requirements](#minimum-requirements)
```

Comment:
Trivial: if there's a sub-point for min reqs, should we also have ones for consuming and building?